### DISCUSSION TITLE
Discarding avoidable dupes of access specifiers

### DISCUSSION THREAD

**Discussion by Anirban166** [nasa/fprime#1567] [General]
Is the use of a duplicate access specifier ([e.g.](https://github.com/nasa/fprime/blob/776af784414b8df7c148845dfca8a1956e5ac4cb/Fw/Types/GTest/Bytes.hpp#L41)) reasonable? ([this](https://stackoverflow.com/a/11362699) seems to be some rationale, but otherwise it seems unnecessary)

**Comment by LeStarch** [comment]
At very least, we allow those.  Most often we break our components (and C++) into sections.

```

// header comment
public:
    //compiler section
//header comment
public:
    //helper functions
...
```

Although that particular example file is missing header comments, the sections are still there.  It is much easier to read and maintain with those extra access specifiers, so you don't have to look outside the section to find what your current specifier is.
